### PR TITLE
Design/home page

### PR DESCRIPTION
**Pull request by IgboPharaoh**
WIP
- [x] Add all necessary page sections to complete desktop design
- [x] make desktop design mobile responsive
- [x] Fix UI copy and nits
- [x] Add granular design details
- [x] Add font family from design 
- [x] Add email js integration

### CODE REVIEW

**Comment by IgboPharaoh** [comment]
Changes for these corrections have been added

File: src/app/components/navbar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  return (
+    <div className='w-full flex items-center justify-center sticky top-0 bg-secondary-black z-50'>
+      <Wrapper className='w-full'>
+        <div className='h-[68px] md:h-[100px] w-full flex items-center justify-between'>
+          <h3 className=' text-xl sm:text-2xl font-extrabold leading-9 tracking-[0.96px]'>
+            MAVAPAY.<span className='text-primary-green'>MONEY</span>
+          </h3>
+          <section className='hidden md:flex items-center gap-5 md:gap-10 text-sm md:text-base'>
+            <Link href='' className='tracking-[0.32px]'>
```
Comment (Extheoisah) [inline src/app/components/navbar/index.tsx:20]: lets link this to the contact us section
```suggestion
            <Link href='#contact-us' className='tracking-[0.32px]'>
```

File: src/app/components/navbar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            <CustomButton label='Sign Up' type='primary' buttonProps={{ style: { padding: "16px 32px" } }} />
+          </section>
+
+          <button className='md:hidden' onClick={() => setOpenMenu(!openMenu)}>
+            {openMenu ? <Cross1Icon height='32px' width='32px' /> : <HamburgerMenuIcon height='32px' width='32px' />}
+          </button>
+        </div>
+        {openMenu ? (
+          <section className='md:hidden flex bg-secondary-black z-50 flex-col absolute top-[68px] right-0 left-0 bottom-0 h-screen overflow-y-scroll overscroll-y-none'>
+            <Link href='' className='tracking-[0.32px] text-white text-xl py-8 px-5 border-t border-b border-input-border'>
```
Comment (Extheoisah) [inline src/app/components/navbar/index.tsx:39]: same here. link to contact us section
```suggestion
            <Link href='#section' className='tracking-[0.32px] text-white text-xl py-8 px-5 border-t border-b border-input-border'>
```

File: src/app/components/say-hello/index.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+                  placeholder: "Email Address",
+                  style: { border: "1.5px solid #DBE1E7", backgroundColor: "#F7F8F9", fontSize: "16px", color: "#090909" },
+                }}
+                className=' placeholder:text-black'
+              />
+              <textarea
+                className='bg-secondary-gray border-[1.5px] py-[26px] px-5 h-[230px] md:h-[300px] border-card-border rounded-lg placeholder:text-secondary-black text-secondary-black'
+                placeholder='Message'
+              />
+              <CustomButton
```
Comment (Extheoisah) [inline src/app/components/say-hello/index.tsx:48]: let's have this send the message to `info@mavapay.co`

**Review by Extheoisah — COMMENTED** [review]
Nice work. left a few comments

**Review by Extheoisah — APPROVED** [review]

**Pull request merged**